cpp/include/cugraph/sampling_functions.hpp

```diff
@@ -1299,29 +1277,28 @@ renumber_and_sort_sampled_edgelist(
  * major key in renumbering and sorting.
  * @param do_expensive_check A flag to run expensive checks for input arguments (if set to `true`).
- * @return Tuple of vectors storing renumbered edge sources, renumbered edge destinations, optional
- * edge weights (valid only if @p edgelist_weights.has_value() is true), optional renumbered edge
- * IDs (valid only if @p edgelist_edge_ids.has_value() is true), optional (label, edge type, hop)
- * offset values to the renumbered and sorted edges (size = @p num_labels * @p num_edge_types * @p
- * num_hops + 1, valid only when @p edgelist_edge_types.has_value(), @p edgelist_hops.has_value(),
- * or @p edgelist_label_offsetes.has_value() is true), renumber_map to query original vertices (size
- * = # unique or aggregate # unique vertices for each label), (label, vertex type) offsets to the
- * vertex renumber map (size = @p num_labels * @p num_vertex_types + 1), optional renumber_map to
- * query original edge IDs (size = # unique (edge_type, edge ID) pairs, valid only if @p
- * edgelist_edge_ids.has_value() is true), and optional (label, edge type) offsets to the edge ID
- * renumber map (size = @p num_labels + @p num_edge_types + 1, valid only if @p
+ * @return Tuple of vectors storing renumbered edge sources, renumbered edge destinations,
+ * edge properties (valid only if @p edgelist_edge_properties.has_value() is true), optional
+ * renumbered edge IDs (valid only if @p edgelist_edge_ids.has_value() is true), optional (label,
+ * edge type, hop) offset values to the renumbered and sorted edges (size = @p num_labels * @p
+ * num_edge_types * @p num_hops + 1, valid only when @p edgelist_edge_types.has_value(), @p
+ * edgelist_hops.has_value(), or @p edgelist_label_offsetes.has_value() is true), renumber_map to
+ * query original vertices (size = # unique or aggregate # unique vertices for each label), (label,
+ * vertex type) offsets to the vertex renumber map (size = @p num_labels * @p num_vertex_types + 1),
+ * optional renumber_map to query original edge IDs (size = # unique (edge_type, edge ID) pairs,
+ * valid only if @p edgelist_edge_ids.has_value() is true), and optional (label, edge type) offsets
+ * to the edge ID renumber map (size = @p num_labels + @p num_edge_types + 1, valid only if @p
  * edgelist_edge_ids.has_value() is true). We do not explicitly return edge source & destination
  * vertex types as we assume that source & destination vertex type are implicilty determined for a
  * given edge type.
  */
 template <typename vertex_t,
-          typename weight_t,
           typename edge_id_t,
           typename edge_type_t>
 std::tuple<
   rmm::device_uvector<vertex_t>,                  // srcs
   rmm::device_uvector<vertex_t>,                  // dsts
-  std::optional<rmm::device_uvector<weight_t>>,   // weights
+  std::vector<arithmetic_device_uvector_t>,       // edge properties
   std::optional<rmm::device_uvector<edge_id_t>>,  // edge IDs
```

Comment:
Do we still need edge IDs? (or can't it be returned as edge properties?)

Reply:
Seems like a better name.  Renamed to `edgelist_edge_additional_properties`